src/components/home/index.js

```diff
@@ -16,9 +21,35 @@ import {
 
 const Home = () => {
   const { showMemberRoleUpdateModal, isSuperUserMode } = userContext();
+  const {
+    state: { newMembers: newMembersList },
+  } = membersContext();
+  const { searchTerm } = searchMemberContext();
+  const filterMembersList = searchMembers(newMembersList, searchTerm);
+
   const { query } = useRouter() || { query: { dev: false } };
   const { dev } = query;
   const [isOptionKey, setIsOptionKey] = useState(false);
+  const [newMembers, setNewMembers] = useState([]);
+  const [scrollCount, setScrollCount] = useState(0);
+
+  const membersToShow = 10;
```

Comment:
Can we please move this to the constants file?

Reply:
Sure!